setup/index.html

```diff
@@ -152,7 +152,7 @@
                 output += "    subscribeKey: '" + data.sub + "'\n";
                 output += "}, {\n";
                 output += "    globalChannel: 'global',\n";
-                output += "    endpoint: 'http://localhost:3000/insecure'\n";
+                output += "    endpoint: 'https://pubsub.pubnub.com/v1/blocks/sub-key/" + data.sub + "/chat-engine-server'\n";
                 output += "});\n";
 
                 $('#code').text(output);
```

Comment:
I think this file could use some refactoring. A DOCTYPE should be added to the top. It is a best practice to move inline JavaScript and CSS to other files. Script elements should be moved to the bottom of the body element. Using a variable to store an element selected by a jQuery selector is a good practice to avoid querying the DOM repeatedly with the jQuery selector(127, 129, 139), and also to make debugging a bit easier. Also with the use of arrow functions already present, perhaps callbacks and Babel would be appropriate too.

Reply:
I just extended what Ian already had here to add the desired functionality and moved on to my next ticket. Sometime after MVP, this script needs entirely re-written to see if apps/keys/functions already exist, have a lot more error handling and take user input, or it's just going to be deleted and have the functionality moved into Portal.

Reply:
I insist on taking this opportunity to do some of the quicker, non-blocking improvements I outlined and also the ones in Ian's comments.